packages/nimble-blazor/NimbleBlazor/Components/NimbleRadioGroup.razor

```diff
@@ -0,0 +1,10 @@
+@namespace NimbleBlazor
+@inherits NimbleInputBase<string>
+<nimble-radio-group disabled=@Disabled
+                    name="@Name"
+                    orientation="@Orientation.ToAttributeValue()"
+                    current-value="@CurrentValue"
```

Comment:
@atmgrifter00 @m-akinc Similar to some of the angular discussions the wrapper for both NimbleRadioGroup and NimbleRadio are exposing properties that bind to the view's value. And both are being exposed to the user so I think they can get out of sync, consider:

```
                <NimbleRadioGroup @ref="group" Name="options">
                    <NimbleRadioButton @ref="radio" Value="1">Option 1</NimbleRadioButton>
                    <NimbleRadioButton Value="2">Option 2</NimbleRadioButton>
                    <NimbleRadioButton Value="3">Option 3</NimbleRadioButton>
                </NimbleRadioGroup>
```

```
group.Value = '1';
radio.checked // probably going to be false right?
```

Do y'all have an idea of how to avoid that? In angular we tried to align with angular's behavior for native radio inputs. Should we do the same and align with blazor's radio input api? https://learn.microsoft.com/en-us/aspnet/core/blazor/forms-and-input-components?view=aspnetcore-5.0#radio-buttons-1
Should we fork it and expose fast's radio button group api as the interface (ie remove the checked property from radio)?

I think I'd lean more towards aligning with the frameworks apis for the native input radio (like we did for Angular) but we may actually need to revisit that more holistically. ie try to get api compatibility between `<NimbleRadioButton>` and `<InputRadio>`

Reply:
I followed the guidance given in our offline conversation. Namely, removing Checked from the radio button API and changing CurrentValue to Value in the radio group.